packages/perseus/src/widgets/interactive-graphs/graphs/segment.tsx

```diff
@@ -49,8 +77,23 @@ const SegmentGraph = (props: SegmentProps) => {
                             ),
                         );
                     }}
+                    ariaLabels={{
+                        point1AriaLabel: formatSegment(
+                            1,
+                            segment[0][X],
+                            segment[0][Y],
+                        ),
+                        point2AriaLabel: formatSegment(
+                            2,
+                            segment[1][X],
+                            segment[1][Y],
+                        ),
+                    }}
                 />
             ))}
-        </>
+            <g id="segment-description" style={{display: "hidden"}}>
```

Comment:
Please use `React.useId()` to create unique IDs. If there are ever multiple segment interactive graphs on the screen, they'd both read the description for the first one.

OHH!!! I JUST FIGURED OUT THE PROBLEM FROM THE ABOVE COMMENT WHILE WRITING THIS!